pkg/util/version/const.go

```diff
@@ -30,44 +30,12 @@ type Stream struct {
 	PullSpec string   `json:"-"`
 }
 
-// DefaultMinorVersion describes the minor OpenShift version to default to
-var DefaultMinorVersion = 11
-
-// DefaultInstallStreams describes the latest version of our supported streams
-var DefaultInstallStreams = map[int]*Stream{
-	10: {
-		Version:  NewVersion(4, 10, 63),
-		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:340091aefa0bba06bbb99cc58cb1f2b73404c832f72b83c526b8e7677efbecef",
-	},
-	11: {
-		Version:  NewVersion(4, 11, 44),
-		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:52cbfbbeb9cc03b49c2788ac7333e63d3dae14673e01a9d8e59270f3a8390ed3",
-	},
-	12: {
-		Version:  NewVersion(4, 12, 25),
-		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:5a4fb052cda1d14d1e306ce87e6b0ded84edddaa76f1cf401bcded99cef2ad84",
-	},
-}
-
-// DefaultInstallStream describes stream we are defaulting to for all new clusters
-var DefaultInstallStream = DefaultInstallStreams[DefaultMinorVersion]
-
-var AvailableInstallStreams = []*Stream{
-	DefaultInstallStreams[10],
-	{
-		Version:  NewVersion(4, 10, 54),
-		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:7e44fa5f6aa15f9492341c4714bba4dc5089c968f2bf77fb8d4cdf189634f8f5",
-	},
-	{
-		Version:  NewVersion(4, 10, 40),
-		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:b9fad814fb4442e7e852b0614d9bb4e2ebc5e1a2fa51623aa838b4ee0e4a5369",
-	},
-	DefaultInstallStreams[11],
-	{
-		Version:  NewVersion(4, 11, 26),
-		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:1c3913a65b0a10b4a0650f54e545fe928360a94767acea64c0bd10faa52c945a",
-	},
-	DefaultInstallStreams[12],
+// Install stream data for production and INT has moved to RP-Config.
```

Comment:
I am wondering, should we also move other image versions to a database?

Reply:
I think it makes sense; it's configuration data.  Downsides are having to repeat the data in each of the four RP-Config cloud files, and it potentially complicates local development mode if it depends on any of those images.